### ISSUE TITLE
Deprecation Warning: Interpolation near operators

### ISSUE THREAD

**Issue by patrickcate** [oddbird/susy#579]
I start a new project using Susy and have been getting the following warning when compiling:

```
Deprecation Warning on line 70 of sass/partials/plugins/susy/susy/output/support/_support.scss: #{}
interpolation near operators will be simplified in a future version of Sass. To preserve the current
behavior, use quotes:

  unquote("#{$_type}-exists")
```

Using Sass: 3.4.21
Using Susy: 2.2.12
Running Sass through [gulp-ruby-sass](https://github.com/sindresorhus/gulp-ruby-sass)

**Comment by mirisuzanne** [comment]
Are you sure you are using 2.2.12? That bug was already fixed in 2.2.10 —https://github.com/oddbird/susy/commit/defabd4bb69c72fe4e3186069b9530d62c4a3122

**Comment by patrickcate** [comment]
I double checked and am using 2.2.12, but I went ahead and deleted the .sass-cache and now the warning is gone so it looks like that was the issue.
